tests/test_adapter.py

```
@@ -294,6 +332,11 @@ async def test_assembly_wires_all_hooks(self, tmp_path: Path, workspace, fake_up
         assert isinstance(fake_upstream["agent_gate"], BudgetAdmission)
         assert hasattr(fake_upstream["observer"], "emit")
         assert callable(fake_upstream["model_resolver"])
+        # worker 基座必须给（缺则上游 backend 逐座位崩——"requires a worker_base_spec"），
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)